src/Artesian/GMEPublicOffers/_Enum/Zone.py

```diff
@@ -2,23 +2,26 @@
 
 
 class Zone(Enum):
-    AUST = 1
-    BRNN = 2
-    CNOR = 3
-    COAC = 4
-    CORS = 5
-    CSUD = 6
-    FOGN = 7
-    FRAN = 8
-    GREC = 9
-    MALT = 10
-    NORD = 11
-    PRGP = 12
-    ROSN = 13
-    SARD = 14
-    SICI = 15
-    SLOV = 16
-    SUD = 17
-    SVIZ = 18
-    CALA = 19
-    MONT = 20
+    AUST = 0
```

Comment:
Please keep the enumeration starting from 1, seems to be like that on all the enums on the Python SDK, probably has to start from 1